web_warning_on_save/README.rst

```diff
@@ -0,0 +1,70 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+web_warning_on_save
```

Comment:
This must be the module title, but in a legible way. For example: `Raise warnings when saving`. The underlining must be of the same size of the title.